docs/portal/src/pages/[tabId].md

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: '[tabId]'
+sidebar_position: 1
+---
+# [tabId]
+
+* js code that shows the whole events page [https://dev.tamudatathon.com/events](https://dev.tamudatathon.com/events)
+* shows the sets, with the cards inside each set
+* can select between the years
```

Comment:
not sure this takes place inside the tabId component, I would think it would be in a parent file. double check please